### DISCUSSION TITLE
Why my contributions are going up and down?

### DISCUSSION THREAD

**Discussion by MitkoVtori** [community/community#35504] [Other Feature Feedback, Questions, & Ideas] [answered]
I had 851 contributions yesterday, when my contributor pinned the project in his profile I had 423, when he unpinned it I had 851 again. And today I have 506! I haven't changed my email, I haven't deleted anything... And it tells me my private contributions are public. Why am I losing contributions?

also, I lost half of my contributions, I had almost 900 and now 484
![Screenshot 2022-10-08 093118](https://user-images.githubusercontent.com/112943652/194693473-9dad117b-4126-4110-a6d7-900232f67147.png)

**Comment by DSDmark** [comment]
#### There is some case it is possibles :-
- you delete any repo of yours, then you lose your contribution 
- Commits made in a fork will not count toward your contributions.
- The commits were made: In the repository’s default branch (usually master) or In the gh-pages branch (for repositories with project sites)
- The email address used for the commits is associated with your GitHub account.

**Reply by MitkoVtori** [reply]
it's none of them. I had this contributions. Yesterday I made a co-authored pull request and get the co-authored achievement badge, but nothing happened to my contributions or commits, that was the last thing I did. After that on the next day I lost half of my commits & contributions. Also, on my contributions graph I twice had more than 100 contributions for the day, now there is written 57, 56 or smth. It could be because I deleted one private repo but it was before 1-2 weeks and nothing happened. I also renamed one project, but not deleted or haven't done anything to it, just rename it. On the picture that I send in the first message you can see it says Hyper Committer, and I am almost Ultra Committer, but I was Ultra Committer and soon I was gonna get the next level.

**Comment by jfullstackdev** [comment, marked as answer]
this might affect that, but I'm not very sure 

![image](https://user-images.githubusercontent.com/47092464/194704456-d55a5834-8b27-4ae8-bf76-e8c8b43f413b.png)

as you can see, you have unverified commits in one of your repos upon checking, that's why that might change from time to time

**Reply by MitkoVtori** [reply]
Thanks, but how can I verify them?

**Reply by Julian-Dumitrascu** [reply]
You can contact GitHub about [this](https://docs.github.com/en/authentication/managing-commit-signature-verification).

**Reply by jfullstackdev** [reply]
depending on your situation,

for example, me, when I'm using GitHub Desktop and I did not install Git, all of the commits from GitHub Desktop are not verified, so what I did was, make a branch on GitHub and instead of pushing directly from GitHub Desktop, it will be through a pull request, that way, it's verified because I'm committing it directly on GitHub 

maybe this will help 

https://docs.github.com/en/authentication/managing-commit-signature-verification

**Reply by MitkoVtori** [reply]
Yea, I saw that when I commit from GitHub everything is verified. Thank you for the idea

**Comment by gift56** [comment]
Hello guys I'm also having the same issues, I got 14,076 commits yesterday on my GitHub profile and today when I refreshed I saw 13,795, I didn't delete, fork, or do anything like that it just went down and I feel it's a bug on GitHub
